zh/asr-wordboosting-zh.ipynb

```diff
@@ -0,0 +1,409 @@
+    "print(\"ASR Transcript with Word Boosting:\", asr_best_transcript)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "047a29cd",
+   "metadata": {},
+   "source": [
+    "#### 3.2 Negative word boosting for undesired words\n",
+    "We can even use word boosting to discourage prediction of some words, by using negative boost scores.  \n",
+    "\n",
+    "For example, "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b7894ae3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating RecognitionConfig\n",
+    "config = riva.client.RecognitionConfig(\n",
+    "  language_code=\"zh-CN\",\n",
+    "  max_alternatives=1,\n",
+    "  audio_channel_count = 1\n",
+    ")\n",
+    "riva.client.add_word_boosting_to_config(config, [\"她\"], -100.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a94fbe96",
+   "metadata": {},
+   "source": [
+    "#### 3.3 Combine negative and positive word boosting\n",
+    "We can also combine the negative word boosting from above example, with positive word boosting. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3995d8e1",
+   "metadata": {},
+   "source": [
+    "Note:\n",
+    "\n",
+    "- There is no limit to the number of words that can be boosted. You should see no impact on latency for all requests, even for ~100 boosted words, except for the first request, which is expected.\n",
+    "- Boosting phrases or a combination of words is not yet fully supported (but do work). We will revisit finalizing this support in an upcoming release.\n",
+    "- By default, no words are boosted on the server side. Only words passed by the client are boosted.\n",
+    "\n",
+    "Information about word boosting can also be found in the documentation [here](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/asr/asr-customizing.html#word-boosting). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64c36fe7",
+   "metadata": {},
+   "source": [
+    "## 4. Go deeper into Riva capabilities\n",
+    "\n",
+    "\n",
+    "### Additional Riva Tutorials\n",
+    "\n",
+    "Checkout more Riva tutorials [here](https://github.com/nvidia-riva/tutorials) to understand how to use some of the advanced features of Riva ASR, including customizing ASR for your specific needs.\n",
+    "\n",
+    "\n",
+    "### Sample Applications\n",
+    "\n",
+    "Riva comes with various sample applications. They demonstrate how to use the APIs to build various applications. Refer to [Riva Sampple Apps](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/samples/index.html) for more information.  \n",
+    "\n",
+    "\n",
+    "### Additional Resources\n",
+    "\n",
+    "For more information about each of the Riva APIs and their functionalities, refer to the [documentation](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/reference/protos/protos.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Looks like there is an extra cell block at the end, could you remove this?

Reply:
Done